### DISCUSSION TITLE
Microsoft’s custom emojis.

### DISCUSSION THREAD

**Discussion by nerddern** [Tarikul-Islam-Anik/Animated-Fluent-Emojis#10] [General]
I can’t find the microsoft teams custom emojis, like the rainbow emoji that is shown in the banner (https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/blob/master/assets/images/banner.gif)
and others custom emojis like it.

**Comment by Tarikul-Islam-Anik** [comment]
Sorry that one is not available on [Emojipedia](https://emojipedia.org/).
